main.py

```diff
@@ -10,6 +11,13 @@ def main():
         lines = read_scores(file_path)
         results = prepare_statistics(lines)
         display_scores(results)
+
+        for student_details in results:
+            username = student_details['username']
+            scores = student_details['scores']
+            student = Student(username, scores)
+            student.display_student_info()
```

Comment:
This for loop should not be needed as it should be implemented within `display_scores`